Sources/BraintreeCoreSwift/BTAnalyticsMetadata.swift

```diff
@@ -0,0 +1,183 @@
+import Foundation
+import UIKit
+
+// TODO: convert to internal (and maybe struct) when BTAnalyticsSession is converted to Swift
+@objcMembers public class BTAnalyticsMetadata: NSObject {
+
+    // MARK: Metadata Properties
+
+    static var platform: String {
+        "iOS"
+    }
```

Comment:
Nope, all of the single line computed properties can be moved to constants, just updated this. Realistically we could just put anything that is on a single line into the dictionary directly, but that could potentially make the dictionary a bit more complicated to reason about, though I don't think we update metadata often so that may be okay. Thoughts?

Reply:
Like this, though metadata doesn't technically need to be computed either:
```
    public static var metadata: [String: Any] {
        [
            "platform": "iOS",
            "platformVersion": UIDevice.current.systemVersion,
            "sdkVersion": BTCoreConstants.braintreeSDKVersion,
            "merchantAppId": Bundle.main.infoDictionary?[kCFBundleIdentifierKey as String] as? String ?? "",
            "merchantAppName": Bundle.main.infoDictionary?[kCFBundleNameKey as String] as? String ?? "",
            "merchantAppVersion": Bundle.main.infoDictionary?[kCFBundleVersionKey as String] as? String ?? "",
            "deviceManufacturer": "Apple",
            "deviceModel": deviceModel,
            "iosDeviceName": UIDevice.current.name,
            "iosSystemName": UIDevice.current.systemName,
            "iosBaseSDK": "\(__IPHONE_OS_VERSION_MAX_ALLOWED)",
            "iosDeploymentTarget": iOSDeploymentTarget,
            "iosIdentifierForVendor": UIDevice.current.identifierForVendor?.uuidString ?? "",
            "iosPackageManager": iOSPackageManager,
            "deviceAppGeneratedPersistentUuid": deviceAppGeneratedPersistentUUID,
            "isSimulator": isSimulator,
            "deviceScreenOrientation": deviceScreenOrientation,
            "venmoInstalled": UIApplication.shared.canOpenURL(URL(string: "com.venmo.touch.v2://x-callback-url/vzero/auth")!),
            "dropinVersion": dropInVersion
        ]
    }
```

Reply:
That could work yeah. We have a [DeviceData](https://github.com/braintree/braintree_android/blob/master/BraintreeCore/src/main/java/com/braintreepayments/api/DeviceMetadata.java) class on Android that captures these values. It uses the builder pattern and has a `toJSON()` method.

Reply:
Probably a bit cleaner to keep the dictionary as is and have the properties at the top. That more similarly matches the android implementation and keeps the dictionary cleaner to read and reason about.